infrastructure/images/moduletest-reset-jobs/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM alpine:3.21.3
```

Comment:
Wouldn't it be simpler to run from a nodejs image like eg `node:22-alpine` or similar? Then you'd only need to install zx.

Reply:
maybe. I was aiming to make as small an image as possible. I can try and make that change and see what happens

Reply:
Did you compare the image sizes? I'm curious about the result.